### PR TITLE
New point release

### PR DESCRIPTION
**Pull request by leonsmith**
Update CHANGELOG and version number for new point release: #22

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 django-s3-storage changelog
 ===========================
 
+0.9.9
+-----
+
+- Added settings for disabling gzip compression (@leonsmith, @bartvds)
+- Bug fix for relative upload paths (@leonsmith)
```
Comment (etianen) [inline CHANGELOG.rst]: Yes, let's do a release! :D

Could you add:

``` rst
- Bug fix for detecting empty directories (@etianen).
- Automatic conversion of windows path separators on upload (@etianen).
```

Also, the general policy here is that credit for features goes to the implementors, not the requester, unless they make a noticeable contribution towards the design. So if you could also remove bartvds from the the credits, that would be good.

(Not trying to be harsh, as requesting features and reporting bugs *is* important, but actually implementing stuff is where the real effort lies, and this is how it's been done to this point)

**Comment by etianen** [comment]
Excellent!

**Comment by etianen** [comment]
0.9.9 is now baked!

**Comment by etianen** [comment]
Huh, no it's not, PyPi is down...

**Comment by leonsmith** [comment]
Awesome and thank you for this lib!

**Pull request merged**